src/Native/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -377,3 +384,11 @@ Returns true/false based on if existing ssl session was re-used or not.
 Shims the SSL_session_reused macro.
 */
 extern "C" bool SslSessionReused(SSL* ssl);
+
```

Comment:
sure (I copied my signature from this actually). I did not realize till now that the size of ```bool``` is implementation defined.